cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -743,16 +784,11 @@ func clusterHasInstance(ctx *controllerContext, instance *compute.Instance, inst
 		return false, err
 	}
 
-	if ctx.csrApproverUseGCEInstanceListReferrers {
-		klog.Infof("using compute.InstancesService.ListReferrers to verify cluster membership of instance %q", instance.Name)
-		ok, err := checkInstanceReferrers(ctx, instance, clusterInstanceGroupUrls)
-		if err != nil {
-			return false, err
-		}
-		if ok {
-			return true, nil
-		}
-		klog.Warningf("could not determine cluster membership of instance %q using compute.InstancesService.ListReferrers; falling back to checking all instance groups", instance.Name)
+	ok, err := checkInstanceReferrersBackOff(ctx, instance, clusterInstanceGroupUrls)
+	if err != nil {
+		return false, err
+	} else if ok {
+		return true, nil
```

Comment:
nit: else if is not required just if will suffice.

Reply:
done.